polaris-react/src/components/Navigation/Navigation.scss

```
@@ -199,6 +224,12 @@ $nav-max-width: 360px;
   #{$se23} & {
     background-color: var(--p-color-bg-app-selected);
 
+    // stylelint-disable-next-line selector-max-combinators -- Needed for precision on text
+    .Text {
+      color: var(--p-color-text);
+      letter-spacing: -0.005rem;
+    }
```

Can we save this number to a localized var that can also be used below?

```suggestion
    .Text {
      --pc-navigation-text-spacing: -0.005rem;
      color: var(--p-color-text);
      letter-spacing: var(--pc-navigation-text-spacing);
    }
```